### ISSUE TITLE
sort-imports rule fails to understand multiple import syntax

### ISSUE THREAD

**Issue by paulbellamy** [eslint/eslint#10878]
**Tell us about your environment**

* **ESLint Version:** 5.6.0
* **Node Version:** 8.9.4
* **npm Version:** 5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?** babel-eslint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  parser: "babel-eslint",
  rules: {
    "sort-imports": [
      "error",
      {
        // this ordering matches one big, fully alphabetical group.
        memberSyntaxSortOrder: [ "none", "all", "single", "multiple" ],
      },
    ],
  },
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import Constants from "./lib/constants";
import { A } from "a";
import { B, C, D } from "bcd";
import { E } from "e";
```

```bash
./node_modules/.bin/eslint foo.js
```

**What did you expect to happen?**

It to succeed, because:
- All the "single" imports (`import Constant ...`) are before the "multiple" imports
- All the "multiple"-syntax imports are sorted

**What actually happened? Please include the actual, raw output from ESLint.**

```
$ ./node_modules/.bin/eslint foo.js

/Users/paulbellamy/temp/eslinttest/foo.js
  2:1  error  Imports should be sorted alphabetically            sort-imports
  4:1  error  Expected 'single' syntax before 'multiple' syntax  sort-imports

✖ 2 problems (2 errors, 0 warnings)
```

The `sort-order` rule thinks that the correct ordering should be:

```
import { A } from "a";
import Constants from "./lib/constants";
import { E } from "e";
import { B, C, D } from "bcd";
```

I believe the bug is that, when there is only one imported field in a "multiple" import syntax, the rule mis-groups it as a "single" import.

**Comment by paulbellamy** [comment]
Ah. That makes sorting imports harder. I was hoping to simply be able to select the import lines and hit "sort" in the editor, since `{` sorts to the end alphabetically.

Thanks anyway.

**Comment by ilyavolodin** [comment]
Closing this issue, as it is working as intended.
